### ISSUE TITLE
Version fails to compile

### ISSUE THREAD

**Issue by GaikwadPratik** [javascript-obfuscator/javascript-obfuscator#623]
## Steps to Reproduce (for bugs)
upgrade to version 1.0.0 and compile

seems like definition is still looking for [TObject](https://github.com/javascript-obfuscator/javascript-obfuscator/blob/master/index.d.ts#L2)

## Your Environment

* Obfuscator version used: `1.0.0`
* Node version used: `10`

# 
Stack trace
```
import { TObject } from './src/types/TObject';
node_modules/javascript-obfuscator/index.d.ts:2:25 - error TS2307: Cannot find module './src/types/TObject' or its corresponding type declarations.
```

P.S. v0.28.1

**Comment by sanex3339** [comment]
Oh, my bad. Will fix it fast

**Comment by sanex3339** [comment]
Fixed in `1.0.1`
